weaviate/collection/classes/config.py

```diff
@@ -1145,23 +1285,44 @@ def to_dict(self) -> Dict[str, Any]:
 
 
 class Property(_ConfigCreateModel):
+    """This class defines primitive data properties, e.g. integer or float, that a collection can have within Weaviate.
```

Comment:
I am not so sure about this. 

When I read `primitive data properties` I expected the class to be the `DataType`. What do you think about `This class defines the structure of a data property that a collection can have within Weaviate.`?